gradle.properties

```diff
@@ -49,4 +49,5 @@ domumornamentum_version=1.20-1.0.150-BETA
 flywheel_version=0.6.8.a-1
 
 # Mod dependencies for testing stuff(Only used in the dev environment)
-jei_version=1.20.1-forge:15.2.0.22
\ No newline at end of file
+jei_version=1.20.1-forge:15.2.0.22
+createadditions_version=5099752
```

Comment:
So is it downgrading `createadditions`? the above is `"curse.maven:createaddition-439890:5099757"`